src/blocks/accordion/edit.js

```diff
@@ -71,44 +70,42 @@ const AccordionEdit = ( props ) => {
 	const items = getBlocksByClientId( clientId );
 
 	const handleEvent = ( ) => {
-		if ( items[ 0 ].innerBlocks ) {
-			const lastId = items[ 0 ].innerBlocks[ items[ 0 ].innerBlocks.length - 1 ].clientId;
-			let copyAttributes = {};
-
-			if ( lastId ) {
-				const lastBlockClient = getBlockAttributes( lastId );
-				if ( lastBlockClient.backgroundColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						backgroundColor: lastBlockClient.backgroundColor,
-					} );
-				}
-
-				if ( lastBlockClient.borderColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						borderColor: lastBlockClient.borderColor,
-					} );
-				}
-
-				if ( lastBlockClient.textColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						textColor: lastBlockClient.textColor,
-					} );
-				}
-
-				if ( lastBlockClient.customTextColor ) {
-					copyAttributes = Object.assign( copyAttributes, {
-						customTextColor: lastBlockClient.customTextColor,
-					} );
-				}
+		const lastId = items[ 0 ].innerBlocks[ items[ 0 ].innerBlocks.length - 1 ]?.clientId;
+		let copyAttributes = {};
+
+		if ( !! lastId ) {
+			const lastBlockClient = getBlockAttributes( lastId );
+			if ( lastBlockClient.backgroundColor ) {
+				copyAttributes = Object.assign( copyAttributes, {
+					backgroundColor: lastBlockClient.backgroundColor,
+				} );
 			}
 
-			const created = createBlock( 'coblocks/accordion-item', copyAttributes );
-			insertBlock( created, undefined, clientId );
+			if ( lastBlockClient.borderColor ) {
+				copyAttributes = Object.assign( copyAttributes, {
+					borderColor: lastBlockClient.borderColor,
+				} );
+			}
+
+			if ( lastBlockClient.textColor ) {
+				copyAttributes = Object.assign( copyAttributes, {
+					textColor: lastBlockClient.textColor,
+				} );
+			}
+
+			if ( lastBlockClient.customTextColor ) {
+				copyAttributes = Object.assign( copyAttributes, {
+					customTextColor: lastBlockClient.customTextColor,
+				} );
+			}
 		}
+
+		const created = createBlock( 'coblocks/accordion-item', copyAttributes );
+		insertBlock( created, undefined, clientId );
```

Comment:
Accordion block tests were broken in previous commits. It looks like it has to do with the fact that `items[ 0 ].innerBlocks` were being interpreted as truthy under any circumstance ( [] == true | [...] == true). This was an issue when on line 73 we checked for the block length without using the `?.` optional selector.

Fixes here allow blocks to be properly inserted when this function fires.